### DISCUSSION TITLE
VC breaking into tech track

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#547] [New Ideas]
With a lot of folks here breaking into tech and based on some of the feedback we've received from the community survey, it would be nice to curate a kind of journey for our members with this role. 

What would this include?

- Resources we find ourselves sharing with new members
- overview of VC-specific things that would appeal to them
- resources like "how to ask a question" or our site resources

If you have thoughts on how to organize this or have resources you think would be useful, reply to this discussion.

**Comment by cafloyd** [comment]
Love this idea! I've collected resources that I  share with early-career devs here: https://www.notion.so/caitlinfloyd/Tips-Resources-for-Early-Career-Devs-3132541c8a594708837e5ba7ae7251a5

I also think highlighting VC-specific things like which channels to join is a great call!

As for how to organize it, if we want it to be something people contribute to ongoingly, a markdown file on the VC repo would be an easy way to do that. Another thought though is some kind of slackbot that DMs new members. It might be nice if it could be spaced out over time so as not to overwhelm them with too much information at once

**Comment by virtualandy** [comment]
Maybe we build more on top [the VC Guide](https://virtualcoffee.io/resources/virtual-coffee/guide-to-vc/). Or make a separate `Software Career Kickstart` microsite.

I like the idea of capturing the various links / tutorials / online courses folks send in a permanent way so they aren't lost in Slack.

A stretch goal could be something like the [Cloud Resume Challenge](https://cloudresumechallenge.dev/). Make a VC specific "New Dev or VC Member Challenge" with goals:
- [ ] attend 3 VC sessions
- [ ] Read the Code of Conduct and "Guide to engaging with the Community" (i.e. how to ask questions)
- [ ] Add or Update a new resource/tutorial/online course
- [ ] Contribute/update to the Challenge itself by adding your name to the list of folks who finished it (maybe you get a special sticker for completing)
